components/CharacterInfo.tsx

```diff
@@ -0,0 +1,87 @@
+function SingleStat({name, value}) {
+  return (
+    <div className="flex">
+      <div className="w-1/2 uppercase">{name}</div>
+      <div className="w-1/2 font-light">{value}</div>
+    </div>
+  )
+}
+
+function StatsOfLevel({label, stats}) {
+  return (
+    <div className="flex-1">
+      <span className="mb-4 text-lg">{label}</span>
+      <div className="flex flex-col">
+        <SingleStat name="Force" value={stats.force}/>
+        <SingleStat name="HP" value={stats.hp}/>
+        <SingleStat name="Attack" value={stats.attack}/>
+        <SingleStat name="Defense" value={stats.defense}/>
+        <SingleStat name="Agility" value={stats.agility}/>
+        <SingleStat name="Crit" value={stats.crit+" %"}/>
+        <SingleStat name="Cdmg" value={stats.cdmg+" %"}/>
+      </div>
+    </div>
+  )
+}
+
+function CharacterInfo({character}) {
+  return (
+    <div className="flex flex-col md:flex-row gap-8 md:p-6 background-repeat"
+      style={{
+        backgroundImage: "url('/background.png')",
+      }}
+      >
+
+      <div className="flex-1 h-auto relative bg-center bg-cover bg-white bg-opacity-25 border-2  flex">
+        <img 
+          className="h-auto my-auto" 
+          src={character.illustration} 
+          alt={`${character.name} (${character.costumeName}) illustration`}/>
+        
+        <span className="flex absolute top-2 left-2">
+          {Array.from({length: character.stars}).map(() =>(
+            <img style={{
+              marginRight: "-8px"
```

Comment:
There is Tailwind classes for negative values : `-mr-2` for 8px
see : https://tailwindcss.com/docs/margin